vnext/Chakra/ChakraExecutor.h

```diff
@@ -69,15 +69,7 @@ class ChakraExecutor : public JSExecutor {
 
   virtual void loadApplicationScript(
       std::unique_ptr<const JSBigString> script,
-#if !defined(OSS_RN)
-      uint64_t scriptVersion,
-#endif
-      std::string sourceURL
-#if !defined(OSS_RN)
-      ,
-      std::string &&bytecodeFileName
-#endif
-      ) override;
+      std::string sourceURL) override;
```

Comment:
>sourceURL [](start = 18, length = 9)

(nit) I realize that currently we are all over the place with casing and code standards, but can we follow Microsoft naming convention about capitalization?
[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)
Should be sourceUrl

Reply:
There is some discussion about this. See [https://github.com/microsoft/react-native-windows/issues/3385](https://github.com/microsoft/react-native-windows/issues/3385).

---
In reply to: [342722760](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342722760) [](ancestors = 342722760)

Reply:
The param name is part of the JSExecutor interface defined in upstream RN. I think it's best to match the parameter name of the original interface.

---
In reply to: [342859594](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342859594) [](ancestors = 342859594,342722760)